cfme/tests/integration/test_openldap_external_auth.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.configure.access_control import Group, User
+from cfme import Credential, login
+from utils.conf import cfme_data
+from utils.providers import setup_a_provider
+
+
+@pytest.fixture(scope="module")
+def setup_first_provider():
+    setup_a_provider(validate=True, check_existing=True)
+
+
+@pytest.fixture()
+def group():
+    data = cfme_data.get("openldap_domain1", {})
+    if not data:
+        pytest.skip("No openldap_test section in yaml")
+    credentials = Credential(
+        principal=data["username"],
+        secret=data["password"],
+    )
+    return Group(description=data['group_name'], role="EvmRole-user",
+                 user_to_lookup=data["username"], ldap_credentials=credentials)
+
+
+@pytest.mark.tier(3)
+@pytest.mark.parametrize(
+    "add_from_ldap", [False, True],
+    ids=['create', 'add_group_from_ldap_lookup'])
+def test_openldap_ext_auth(request, group, add_from_ldap, configure_openldap_external_auth_mode):
+    data = cfme_data.get("openldap_domain1", {})
```

Comment:
As you have the data here, and above. It probably makes sense to make the User a fixture too, and then make the data a fixture that is required by both User and Group.